File: socialuni-server/socialuni-common-api/src/main/java/com/socialuni/social/common/api/model/SocialuniPageQueryQO.java
package com.socialuni.social.common.api.model;

import lombok.Data;

import java.util.Date;

@Data
public class SocialuniPageQueryQO<T> {
    private Boolean firstLoad;
    private Date queryTime;
    private Integer pageSize;
    private Integer pageNum;
    private T queryData;
}

File: socialuni-js/packages/socialuni-api/socialuni-api-base/src/model/common/SocialuniPageQueryQO.ts
import SocialuniContentRO from "../social/SocialuniContentRO";
import LoadMoreType from "socialuni-constant/constant/LoadMoreType";

export default class SocialuniPageQueryQO<Q> {
    firstLoad = true
    queryTime = new Date()
    pageSize = 10
    pageNum = 1
    queryData: Q = null
}
